spring-rabbit-test/src/main/java/org/springframework/amqp/rabbit/test/RabbitListenerSpyTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.amqp.rabbit.test;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.amqp.rabbit.annotation.EnableRabbit;
+import org.springframework.context.annotation.Import;
+
+/**
+ * Annotate a {@code @Configuration} class with this to enable Mockito
+ * spying on {@code @RabbitListener} beans to validate arguments.
+ *
+ * @author Gary Russell
+ * @since 1.6
+ *
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@EnableRabbit
+@Import(RabbitListenerSpyTestBootstrap.class)
+public @interface RabbitListenerSpyTest {
```

Comment:
That's what I did initially but the bootstrap class doesn't have access to the annotation.

If there's another way, I'm open to it.